packages/strapi-design-system/src/JSONInput/JSONInput.tsx

```diff
@@ -93,14 +98,19 @@ export const JSONInput = ({ label, value, error, hint, required, onChange, disab
       syntaxHighlighting: true,
       highlightSelectionMatches: true,
       tabSize: 2,
-      defaultCharacterWidth: 5,
     },
+    container: editor.current,
+    editable: !disabled,
+    extensions: [json(), markField],
+    onChange: onCodeMirrorChange,
+    theme: 'dark',
   });
 
   const focusInput = () => {
     if (!disabled) {
       // Focus the content editable element nested in the JSONInputContainer ref
-      editor.current.children[0].children[1].children[1].focus();
+      const contentEditable = editor.current?.children[0].children[1].children[1] as HTMLElement;
+      contentEditable.focus();
```

Comment:
I was getting `Element` instead of what I expected: `HTMLElement`. I've type casted it here now to include null. I'm not sure though, maybe there is a better solution?